{"name": "test_failed_login[invalid_user-wrong_pass-Epic sadface: Username and password do not match any user in this service]", "status": "passed", "steps": [{"name": "Test invalid login: invalid_user/wrong_pass", "status": "passed", "start": 1754369123077, "stop": 1754369123576}], "attachments": [{"name": "log", "source": "6021ee16-787c-4d1c-8c1d-5028103aa0db-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "6acb8b3d-d67d-4219-8577-fd95f1cdd949-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'invalid_user'"}, {"name": "password", "value": "'wrong_pass'"}, {"name": "error_message", "value": "'Epic sadface: Username and password do not match any user in this service'"}], "start": 1754369123076, "stop": 1754369123576, "uuid": "1ef1e5c3-18ec-461d-9a4c-9a4dfcf50fdf", "historyId": "009e9186f369e24c9a13e6a3a9bf681d", "testCaseId": "eb98303b78b0dd8a81a7d4c9688110a7", "fullName": "tests.test_login#test_failed_login", "labels": [{"name": "tag", "value": "regression"}, {"name": "story", "value": "Failed Login"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "LAPTOP-E3GPSS8Q"}, {"name": "thread", "value": "8468-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}